apps/builder/app/builder/features/style-panel/style-layer-utils.ts

```diff
@@ -41,28 +41,34 @@ export const deleteLayer = <T extends TupleValue | LayersValue>(
   batch.publish();
 };
 
-export const hideLayer = (
+export const hideLayer = <T extends LayersValue | TupleValue>(
   property: StyleProperty,
   index: number,
-  layers: LayersValue,
+  layers: T,
```

Comment:
Please use a full word always when possible, here clearly it could be 
```suggestion
  layers: Layers,
```